msteams-platform/toolkit/configure-bot-capability.md

```diff
@@ -0,0 +1,522 @@
+
+   ```json
+    {
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "resourceBaseName": {
+        "value": "tab${{RESOURCE_SUFFIX}}"
+      },
+      "webAppSku": {
+        "value": "B1"
+      },
+      "botAadAppClientId": {
+        "value": "${{BOT_ID}}"
+      },
+      "botAadAppClientSecret": {
+        "value": "${{SECRET_BOT_PASSWORD}}"
+      },
+      "botDisplayName": {
+        "value": "bot"
+      },
+      "resourceBaseName2":{
+        "value": "bot${{RESOURCE_SUFFIX}}"
+      }
+    }
+    }
+   ```
+
+1. Add `botAadApp/create` action under `provision` section in the `teamsapp.yml` file.
+1. Add the following code in `deploy` section:
+
+   ```yml
+    deploy:
+      - uses: cli/runNpmCommand # Run npm command
+        with:
+          args: install
+      - uses: cli/runNpmCommand # Run npm command
+        with:
+          args: run build
+      # Deploy bits to Azure Storage Static Website
+      - uses: azureAppService/zipDeploy
+        with:
+          workingDirectory: ./tab
+          # Deploy base folder
+          artifactFolder: .
+          # Ignore file location, leave blank will ignore nothing
+          ignoreFile: .webappignore
+          # The resource id of the cloud resource to be deployed to.
+          # This key will be generated by arm/deploy action automatically.
+          # You can replace it with your existing Azure Resource id
+          # or add it to your environment variable file.
+          resourceId: ${{TAB_AZURE_APP_SERVICE_RESOURCE_ID}}
+      - uses: azureAppService/zipDeploy
+        with:
+          workingDirectory: ./bot
+          # Deploy base folder
+          artifactFolder: .
+          # Ignore file location, leave blank will ignore nothing
+          ignoreFile: .webappignore
+          # The resource id of the cloud resource to be deployed to.
+          # This key will be generated by arm/deploy action automatically.
+          # You can replace it with your existing Azure Resource id
+          # or add it to your environment variable file.
+          resourceId: ${{BOT_AZURE_APP_SERVICE_RESOURCE_ID}}
+   ```
+
+   For more information, see the [sample project](https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/hello-world-bot-with-tab).
+
+1. Run `Teams: Provision` command to apply the bicep to Azure.
+1. Run `Teams: Deploy` command to deploy your app code to Azure.
+1. Open the **Run and Debug Activity Panel** and select **Launch Remote (Edge)** or **Launch Remote (Chrome)**.
+1. Select F5 key to debug and preview your Teams app.
+
+# [Add bot to Teams message extension app](#tab/messageextensionapp)
+
+You can add a message extension to a Teams bot app more easily than adding it to a tab app, as message extensions use the bot support framework.
```

Comment:
This should be vice versa

You can add a Teams bot to a message extension app more easily than adding it to a tab app,....